FL/x.H

```diff
@@ -82,7 +82,8 @@ typedef ulong Fl_Offscreen;
   fl_push_no_clip()
 #    define fl_end_offscreen() \
   fl_pop_clip(); fl_window = _sw; _ss->set_current(); \
-  if (!_sgc) XFreeGC(fl_display, fl_gc); fl_gc = _sgc
+  if (!_sgc) XFreeGC(fl_display, fl_gc); \
+  fl_gc = _sgc
```

Comment:
This looks like an acceptable - although unnecessary - change.

It's unnecessary because the warning is benign.

Reply:
Fixed anyway as proposed.